ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -63,7 +63,8 @@ Test Inference With Token Authentication
     [Documentation]    Test the inference result after having deployed a model that requires Token Authentication
     [Tags]    Sanity    Tier1
     ...    ODS-1920
-    Run Keyword And Continue On Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT}    ${EXPECTED_INFERENCE_OUTPUT}    token_auth=${TRUE}
+    # Run Keyword And Continue On Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT}    ${EXPECTED_INFERENCE_OUTPUT}    token_auth=${TRUE}
```

Comment:
Do we want to keep this line here?

Reply:
actually yes, I wanted to keep it because the new line is a tentative and temporary fix, it should be reverted back when a better solution is find (inference requests failing intermittently, seems timing but need deeper investigation)